api/database/factories/PoolCandidateFactory.php

```diff
@@ -59,12 +59,7 @@ public function definition()
                 ],
                 3
             ),
-            'pool_candidate_status' => $this->faker->randomElement([
-                'AVAILABLE',
-                'PLACED_INDETERMINATE',
-                'PLACED_TERM',
-                'NO_LONGER_INTERESTED',
-            ]),
+            'pool_candidate_status' => $this->faker->boolean() ? ApiEnums::CANDIDATE_STATUS_AVAILABLE :ApiEnums::candidateStatuses()[array_rand((ApiEnums::candidateStatuses()))],
```

Comment:
This line is quite hard to read.  Could you please reformat it?

Reply:
[hopefully this helps](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3143/commits/1c8c6018a4bbe347c7742a77b3883235ffa39717)